modules/pel/peltool/peltool.py

```diff
@@ -245,6 +245,42 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
```

Comment:
I feel, commit 1 and 2 could be squashed and then commit 3 and 4 could be squashed.

Reply:
yea, lets get the 2 commits down to 1.